src/services/agent-service.ts

```diff
@@ -107,13 +107,6 @@ export class AgentService {
     }
 
     let domSnapshot = request.body.domSnapshot
-    const percyCSSFileName = `percy-specific.${Date.now()}.css` as string
-
-    // Inject the link to the percy specific css if the option is passed
-    if (snapshotOptions.percyCSS) {
-      const cssLink = `<link data-percy-specific-css rel="stylesheet" href="/${percyCSSFileName}" />`
-      domSnapshot = domSnapshot.replace(/<\/body>/i, cssLink + '$&')
-    }
 
     if (domSnapshot.length > Constants.MAX_FILE_SIZE_BYTES) {
```

Comment:
Is this limit a hard limit, or our own? If it's a hard limit, we might want to pull the `cssLink` var to above this so it can be used in the length calculation. `if ((domSnapshot.length + cssLink.length) > .....`

Reply:
This is our own limit, just so logs weren't massive. I almost feel like we can _remove_ this now that we're collecting logs in a much different way. 

We did this because the DOM snapshot string would be MASSIVE and having people copy paste the logs became pretty hard.

Reply:
If it isn't a hard limit, then it shouldn't matter that 80-100 bytes are added